library/src/main/scala/scala/scalajs/js/BigInt.scala

```diff
@@ -0,0 +1,188 @@
+  def ==(x: Float): Boolean = js.native
+  def ==(x: Double): Boolean = js.native
+
+  def !=(x: BigInt): Boolean = js.native
+  def !=(x: Byte): Boolean = js.native
+  def !=(x: Short): Boolean = js.native
+  def !=(x: Char): Boolean = js.native
+  def !=(x: Int): Boolean = js.native
+  def !=(x: Long): Boolean = js.native
+  def !=(x: Float): Boolean = js.native
+  def !=(x: Double): Boolean = js.native
+
+  def <(x: BigInt): Boolean = js.native
+  def <(x: Byte): Boolean = js.native
+  def <(x: Short): Boolean = js.native
+  def <(x: Char): Boolean = js.native
+  def <(x: Int): Boolean = js.native
+  def <(x: Long): Boolean = js.native
+  def <(x: Float): Boolean = js.native
+  def <(x: Double): Boolean = js.native
+
+  def <=(x: BigInt): Boolean = js.native
+  def <=(x: Byte): Boolean = js.native
+  def <=(x: Short): Boolean = js.native
+  def <=(x: Char): Boolean = js.native
+  def <=(x: Int): Boolean = js.native
+  def <=(x: Long): Boolean = js.native
+  def <=(x: Float): Boolean = js.native
+  def <=(x: Double): Boolean = js.native
+
+  def >(x: BigInt): Boolean = js.native
+  def >(x: Byte): Boolean = js.native
+  def >(x: Short): Boolean = js.native
+  def >(x: Char): Boolean = js.native
+  def >(x: Int): Boolean = js.native
+  def >(x: Long): Boolean = js.native
+  def >(x: Float): Boolean = js.native
+  def >(x: Double): Boolean = js.native
+
+  def >=(x: BigInt): Boolean = js.native
+  def >=(x: Byte): Boolean = js.native
+  def >=(x: Short): Boolean = js.native
+  def >=(x: Char): Boolean = js.native
+  def >=(x: Int): Boolean = js.native
+  def >=(x: Long): Boolean = js.native
+  def >=(x: Float): Boolean = js.native
+  def >=(x: Double): Boolean = js.native
+
+  /**
+   *
+   * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toLocaleString
+   * @param locale A string with a BCP 47 language tag, or an array of such
+   *               strings. For the general form and interpretation of the
+   *               locales argument, see the Intl page.
+   * @param options An object with some or all of the properties.
+   * @return
+   */
+  def toLocaleString(locale: String, options: BigIntOptions = js.native): String = js.native
+
+  /**
+   * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toString
+   * @return A string representing the specified BigInt object.
+   *         The trailing "n" is not part of the string.
+   */
+  override def toString(): String = js.native
+
+  /**
+   * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/valueOf
+   * @return The wrapped primitive value of a BigInt object.
+   */
+  override def valueOf(): BigInt = js.native
+}
+
+/**
+ * An object with some or all of the following properties.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toLocaleString#Parameters
+ */
+trait BigIntOptions extends Object {
```

Comment:
This should be inside the `object BigInt` and be called `ToLocaleStringOptions`. Those options are very specific to `toLocaleString`.